lib/resources/app_strings.dart

```diff
@@ -1,9 +1,10 @@
-
 class AppStrings {
```

Comment:
Make `AppString` class abstract and remove the private constructor

Reply:
Done